guard/src/commands/validate/tf.rs

```diff
@@ -55,22 +51,20 @@ impl<'reporter> Reporter for TfAware<'reporter> {
     ) -> crate::rules::Result<()> {
         let root = data.root().unwrap();
         let is_tf_plan = match data.at("/resource_changes", root) {
-            Ok(_resource_changes) => match data.at("/terraform_version", root) {
-                Ok(_tf_version) => true,
-                _ => false,
-            },
+            Ok(_resource_changes) => matches!(data.at("/terraform_version", root), Ok(_tf_version)),
             _ => false,
         };
 
         if is_tf_plan {
             let failure_report = simplifed_json_from_root(root_record)?;
-            Ok(match output_type {
+            match output_type {
                 OutputFormatType::YAML => serde_yaml::to_writer(write, &failure_report)?,
                 OutputFormatType::JSON => serde_json::to_writer_pretty(write, &failure_report)?,
                 OutputFormatType::SingleLineSummary => {
                     single_line(write, data_file, rules_file, data, root, failure_report)?
                 }
-            })
+            };
+            Ok(())
```

Comment:
Same question as before: are we changing the return type?

Reply:
same answer., we're not changing the return type. But this is exactly why its best practice to return unit types this way